### PR TITLE
improve custom token dialog

### PR DESCRIPTION
**Pull request by chen-yijiang**
#1804

image

### CODE REVIEW

File: src/shared/wallet/add-custom-tokens-form-modal.tsx [hunk 1]
```diff
@@ -65,7 +84,7 @@ class AddCustomTokensFormModal extends React.PureComponent<
           >
             <div style={{ width: 180, marginRight: 20 }}>
               <img
-                src={`/image/token/${item.logo}`}
+                src={`https://iotexscan.io/image/token/${item.logo}`}
```
Comment (anshuomomo) [inline src/shared/wallet/add-custom-tokens-form-modal.tsx:87]: The domain should not hard code i think.

File: src/shared/wallet/add-custom-tokens-form-modal.tsx [hunk 2]
```diff
@@ -93,7 +117,7 @@ class AddCustomTokensFormModal extends React.PureComponent<
         }}
       >
         <Form.Item
-          label={t("wallet.input.fromXrc20")}
+          label={t("wallet.input.search")}
```
Comment (anshuomomo) [inline src/shared/wallet/add-custom-tokens-form-modal.tsx:120]: I saw the label 'Search' is now float right in the picture. Does it in the right place? Please check.

File: src/shared/wallet/add-custom-tokens-form-modal.tsx [hunk 3]
```diff
@@ -4,7 +4,7 @@ import Modal from "antd/lib/modal";
 
 // @ts-ignore
 // tslint:disable-next-line:import-blacklist
-import { AutoComplete, Input } from "antd";
+import {AutoComplete, Input, message} from "antd";
```
Comment (anshuomomo) [inline src/shared/wallet/add-custom-tokens-form-modal.tsx:7]: Did the 'message' import used here?

File: src/shared/wallet/add-custom-tokens-form-modal.tsx (hunk 2 above)
Comment (chen-yijiang) [reply src/shared/wallet/add-custom-tokens-form-modal.tsx:120]: I have not modified the location of the label，If necessary, I suggest creating an issue.

File: src/shared/wallet/add-custom-tokens-form-modal.tsx (hunk 3 above)
Comment (chen-yijiang) [reply src/shared/wallet/add-custom-tokens-form-modal.tsx]: done

File: src/shared/wallet/add-custom-tokens-form-modal.tsx (hunk 1 above)
Comment (chen-yijiang) [reply src/shared/wallet/add-custom-tokens-form-modal.tsx]: done

File: src/shared/wallet/add-custom-tokens-form-modal.tsx (hunk 2 above)
Comment (anshuomomo) [reply src/shared/wallet/add-custom-tokens-form-modal.tsx:134]: Please help to fix it to the right position in this PR.

Comment (chen-yijiang) [reply src/shared/wallet/add-custom-tokens-form-modal.tsx:134]: done

WeChatdf8b9777a4604ebc3d4c673ed9d1c718

**Review by anshuomomo — APPROVED** [review]

**Pull request merged**